### PR TITLE
informative delete pages + html tags checklist

### PR DESCRIPTION
**Pull request by shaySirek**
- **Improve ingredient delete page**

image

image

- **Improve user delete page**

image

image

### CODE REVIEW

**Comment by shaySirek** [comment]
`GetMap` minor fix:
- avoid `store.point undefined`
- center is calculated only by valid points (`store.point` is defined, `api` call worked)

**Comment by shaySirek** [comment]
[**Clock with `canvas`**](https://github.com/InternetApp2020/InternetApp2020/pull/25/commits/f942aec4a4971f644a6bd751a1f042ab6f628992)

image

File: Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -43,30 +41,118 @@
...
     <script src="~/js/jquery-3.5.0.min.js"></script>
     <script src="~/js/bootstrap.min.js"></script>
     <script src="~/js/site.js"></script>
 
+    <script>
+        var canvas = $('#clock').get(0);
+        var ctx = canvas.getContext("2d");
+        var radius = canvas.offsetWidth / 2;
+        ctx.translate(radius, radius);
+        radius = radius * 0.90
```
Comment (omerzamir) [inline Views/Shared/_Layout.cshtml:79]: missing semicolon

**Comment by shaySirek** [comment]
https://github.com/InternetApp2020/InternetApp2020/pull/25/commits/93f095f9470088e5a193506c31403622e5b0b983: different html tag, looks the same :)

**Comment by shaySirek** [comment]
https://github.com/InternetApp2020/InternetApp2020/pull/25/commits/a41715740f411fc47153c4f914a2bbf94de77312
------------------------------------------
image

**Pull request merged**